web-platform-tests/wpt.fyi
# Add feature for exposing a compare button and /analyzer endpoint

## Description
Adds a /analyzer endpoint that takes a repeated `screenshot` param (or, `before`/`after`) of the screenshot `[HashMethod]:[HashDigest]` identifiers.

## Review Information
- Visit /flags
- Enable Reftest Analyzer
- Enable the mock data too
- Visit any reftest
- Hit the `Compare` button
- See a reftest analyzer with the mock screenshots
